client/src/components/Button.jsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+const Button = ({ title, type, myFunction }) => {
+  return (
+    <DefaultButton type={type} onClick={myFunction}>
+      {title}
+    </DefaultButton>
+  );
+};
+
+export default Button;
+
+const DefaultButton = styled.button`
+  background-color: inherit;
+  border: none;
+  background-color: var(--color-cyan);
+  border-radius: var(--border-radius);
+  height: 3rem;
+  width: 100%;
+  cursor: pointer;
```

Comment:
I think it is really cool that you think about your non mobile version of your app, but I just want to mention, that your focus should lie on mobile first. Just as a reminder. :)